projects/1.18/assets/dynamic-electricity/dynamicelectricity/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+  "block.dynamicelectricity.motorachv": "480V交流电动机",
+  "block.dynamicelectricity.motoraclv": "120V交流电动机",
+  "block.dynamicelectricity.motoracmv": "240V交流电动机",
+  "block.dynamicelectricity.motordchv": "480V直流电动机",
+  "block.dynamicelectricity.motordclv": "120V直流电动机",
+  "block.dynamicelectricity.motordcmv": "240V直流电动机",
+  "container.motorachv": "480V交流电动机",
+  "container.motoraclv": "120V交流电动机",
+  "container.motoracmv": "240V交流电动机",
+  "container.motordchv": "480V直流电动机",
+  "container.motordclv": "120V直流电动机",
+  "container.motordcmv": "240V直流电动机",
+  "fluid.dynamicelectricity.fluidlubricant": "工业润滑油",
+  "gui.dynamicelectricity.displayunit.industrialenergyunit.name": "工业能量单元",
+  "gui.dynamicelectricity.displayunit.industrialenergyunit.nameplural": "工业能量单元",
+  "gui.dynamicelectricity.displayunit.industrialenergyunit.symbol": "IE",
+  "gui.dynamicelectricity.motor.generating": "生成中",
+  "gui.dynamicelectricity.motor.lubricant": "润滑油：",
+  "gui.dynamicelectricity.motor.output": "输出：%s",
+  "gui.dynamicelectricity.motor.tier": "等级：%s",
+  "gui.dynamicelectricity.motor.usage": "使用：%s",
+  "gui.dynamicelectricity.motor.voltage": "电压：%s",
+  "gui.dynamicelectricity.motor.wattage": "功率：%s",
+  "guidebook.dynamicelectricity.chapter.acmotors": "交流电动机",
+  "guidebook.dynamicelectricity.chapter.acmotors.l1": "交流电动机能够将电动力学中的焦耳（J）以100%的效率转换为Forge能量单位（FE）。电动机有三个等级，分别提供不同的吞吐率以满足你的需求。",
+  "guidebook.dynamicelectricity.chapter.dcmotors": "直流电动机",
+  "guidebook.dynamicelectricity.chapter.dcmotors.l1": "直流电动机能够将Forge能量单位（FE）以95%的效率转换为电动力学中的焦耳（J）。与交流电动机不同，直流电动机需要一个名为导体刷的额外组件来运行。刷子的耐久度有限，但可以通过从电动机顶部使用漏斗来轻松补充。电动机有三个等级，分别提供不同的吞吐率以满足你的需求。",
+  "guidebook.dynamicelectricity.chapter.dynamicelectricity": "Dynamic Electricity",
+  "guidebook.dynamicelectricity.chapter.dynamicelectricity.l1": "Dynamic Electricity面向大规模能源转换。电动力学的电池箱可以自然地将FE转换为焦耳。它们也能够为FE机器供电，但仅限于FE机器的电压额定为120V。然而，电池箱的输入和输出速率有限。Dynamic Electricity提供了一系列电动机，其吞吐量值远高于电池箱的能力。然而，虽然电动机提供了更高的吞吐量，但它们需要多加保养才能运作，包括提供润滑油来保持其运行。幸运的是，润滑油的制造成本相当低，可以简单地通过管道输送到电动机底部。电动机也会持续转换能量，不论它是否能被使用。但是幸好，它们可以通过红石信号控制开关，这意味着你可以在需要时控制它们是否开启。",
```

```suggestion
  "guidebook.dynamicelectricity.chapter.dynamicelectricity.l1": "Dynamic Electricity旨在解决大规模能源转换问题。尽管电动力学的电池箱可以自行将FE转换为焦耳，也能够为额定电压为120V的FE机器供电，但输入和输出速率有限。Dynamic Electricity提供了一系列电动机，提供远高于电池箱的吞吐量值。然而，虽然电动机提供了更高的吞吐量，但它们需要多加保养才能运作，包括提供润滑油来保持其运行。幸运的是，润滑油的制造成本相当低，可以简单地通过管道输送到电动机底部。不论是否使用，电动机都将持续转换能量。不过幸运的是，它们可以通过红石信号开关，这意味着你可以根据需要控制它们的启停。",
```
1. geared towards 指某个事物或活动是为了满足某个特定目标或人群的需求而设计或调整的。在此需要灵活翻译，不能一味译作“面向”。
2. can naturally 虚指，务必不要逐字译出。
3. 2、3、4句主语一致，句意连贯，合并为一个句子。
4. capable of 能达到的、能够做到的。此处原译有语病，吞吐量值和能力不是同类事物，不可比较。应当说吞吐能力。直译应当是“提供了远高于电池箱所能提供的吞吐量值”。
5. 看见whether首先想到调整语序。regardless of强调，忽略即可。
> I'm going whether you like it or not. 不管你愿意不愿意，我都要走了。
6. 仍然存在一些翻译腔问题。长文本尤其要求理解和再创作，同时兼顾准确性。

---

感谢指正🙇‍我再去看看其它条目